tutorials/proto_rename/PersonTool.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2018 The StartupOS Authors.
```

Comment:
I think it's easier to get around one file, and having multiple files adds boilerplate. How about we merge `PersonReader.java`, `PersonWriter.java` and `ProtoRenameComponent.java` all into `PersonTool.java`?

Reply:
Done.